macros/identifiers.sql

```diff
@@ -0,0 +1,239 @@
+{#
+Copyright (c) 2022-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+{% macro session_identifiers() %}
+  {{ return(adapter.dispatch('session_identifiers', 'snowplow_media_player')()) }}
+{% endmacro %}
+
+
+{% macro default__session_identifiers() %}
+
+  {% if var('snowplow__session_identifiers') %}
+    {{ return(var('snowplow__session_identifiers')) }}
```

Comment:
I'm not sure how I feel about `databricks/spark` being the default. Maybe it would be cleaner (and good practise going forward) to make it so that if default is only warehouse (or pair of adapters) to set the default as a not supported warning, and deifne each one specifically? I'm not that bothered, maybe just add a comment actually?

Reply:
Yeah the defining them individually approach sounds better. Especially as it's it was more common for redshift/postgres to be the default in other packages, this being databricks/spark is going to end up confusing